docs/conf.py

```diff
@@ -26,6 +26,16 @@
 
 master_doc = 'docs/index'
 
+# Versions to show in the version menu
+# NOTE: we could be smarter and detect these from branch/tag names,
+# for example
+versions = [
+        'master',
+        'v0.4'
```

Comment:
@marquiz We need to have "human readable labels" for the branches. E.g. "master" -> "development", "release-v0.4" -> "stable" and "latest".

Reply:
Changed in the latest version:
- add symlink `stable` to the latest release
- use name `devel` for master branch